lib/fibered_mysql2/hash_config_override.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module FiberedMysql2
+  module HashConfigOverride
+    # Override the reaping_frequency method to return nil so that the connection pool does not reap connections when in fibered mode.
+    def reaping_frequency
```

Comment:
We can no longer modify `reaping_frequency` dynamically within the db config, so this just overrides the reaping frequency completely.